repo/packages/K/kubernetes/100/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "1.0.0-1.9.3"
```

Comment:
@pires @bmcstdio shouldn't it be `1.0.1-1.9.3` ?

Reply:
We've only released 1.0.0-1.9.3

https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kubernetes/0/package.json#L5

Reply:
@hectorj2f yeah, @benclarkwood's right.

Reply:
@benclarkwood Thanks for the clarification